dags/services/AWSService.py

```diff
@@ -0,0 +1,40 @@
+import boto3
+from airflow.models import Variable
+from pathlib import Path
+
+# AWS creds
+AWS_CREDS = {
+    "AWS_ACCESS_KEY_ID": Variable.get("TOWER_DB_ACCESS_KEY"),
+    "AWS_SECRET_ACCESS_KEY": Variable.get("TOWER_DB_SECRET_ACCESS_KEY"),
+}
+# AWS region
+AWS_REGION = "us-east-1"
+
+def initialize_aws_client(resource: str, aws_creds: dict = AWS_CREDS, aws_region: str = AWS_REGION):
```

Comment:
I think as an initial iteration, these are good, but what are your thoughts of using existing operators so we don't need to maintain our own: https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/operators/s3.html?